examples/volume.go

```diff
@@ -12,9 +12,8 @@ import (
 func main() {
 	groupService, err := service.NewNimbleGroupService(
 		service.WithHost("1.1.1.1"),
-		service.WithUser("xxx"),
-		service.WithPassword("xxx"),
-		service.WithoutWaitForAsyncJobs())
```

Comment:
Is the change to asynchronous jobs intentional?

Is there an example anywhere that shows how to use the SDK to process an async job?  I.e., to interpret the response and recognize that the job is incomplete and then poll for its completion.  I would love to see that.

For now this change to use synchronous wait is probably a good idea.  I suggest adding a separate item to the backlog to provide an async job example, if reasonably feasible.

Reply:
@raunakkumar, thoughts about adding a backlog item to create an async job example?